### PR TITLE
Add new Laos Schools overlays and move old ones

### PR DESCRIPTION
**Pull request by Alastair-L**
### Issue #:
https://github.com/beyondessential/tupaia-backlog/issues/1054
### Changes:
Very similar to: https://github.com/beyondessential/tupaia-backlog/issues/936

### CODE REVIEW

File: packages/database/src/migrations/20200914013941-AddTextbookStudentRatioOverlaysDistrictAndProvinceLevel-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+};
+
+const mapToLevel = (overlays, level) =>
+  overlays.map(overlayConfig => ({
+    ...overlayConfig,
+    measureLevel: level,
+    id: `${overlayConfig.baseId}_${level}`,
+  }));
+
+const OVERLAYS_IN_HIERARCHY = {
```
Comment (Alastair-L) [inline packages/database/src/migrations/20200914013941-AddTextbookStudentRatioOverlaysDistrictAndProvinceLevel-modifies-data.js:308]: Note that this structure is not what was asked for in the card and may need to be altered.

Comment (Alastair-L) [reply packages/database/src/migrations/20200914013941-AddTextbookStudentRatioOverlaysDistrictAndProvinceLevel-modifies-data.js:308]: Has been approved in the card. Resolving 🎉

File: packages/database/src/migrations/20200914013941-AddTextbookStudentRatioOverlaysDistrictAndProvinceLevel-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+  if (Array.isArray(children)) {
+    // Children are map overlays. Base case
+    return Promise.all(
+      children.map((overlay, index) => insertOverlay(db, overlay, index, groupId)),
+    );
+  }
+  // Children are map overlay groups. Recurse.
+  return Promise.all(
+    Object.entries(children).map(([newGroup, newChildren]) =>
+      insertGroupAndChildren(db, newGroup, newChildren, groupId),
```
Comment (nick-beyondessential) [inline packages/database/src/migrations/20200914013941-AddTextbookStudentRatioOverlaysDistrictAndProvinceLevel-modifies-data.js:576]: Nice :)

**Review by nick-beyondessential — APPROVED** [review]
Epic migration! Looks clean to me. Nice use of comments to clarify.

**Pull request merged**